R/html_print.R

```diff
@@ -71,6 +71,11 @@ html_print <- function(html, background = "white", viewer = getOption("viewer",
 #'
 #' @export
 save_html <- function(html, file, background = "white", libdir = "lib", lang = "en") {
+  UseMethod("save_html")
+}
+
+#' @export
+save_html.default <- function(html, file, background = "white", libdir = "lib", lang = "en") {
```

Comment:
It's not strictly necessary, but we might want to add `...` into the function signature to allow for argument expansion in the methods.

```suggestion
save_html <- function(html, file, background = "white", libdir = "lib", lang = "en", ...) {
  UseMethod("save_html")
}

#' @export
save_html.default <- function(html, file, background = "white", libdir = "lib", lang = "en", ...) {
```